speech/api/README.md

```diff
@@ -48,15 +48,17 @@ Before running these samples perform the steps:
     cd python-docs-samples/speech/api
     ```
 
-* Create virtualenv
+* Create a [virtualenv][virtualenv]
     ```
     virtualenv env
     source env/bin/activate
     ```
 
 ### Install the dependencies
 
-The sample uses the [PyAudio][pyaudio] library to stream audio from your computer's microphone.  PyAudio depends on [PortAudio][portaudio], which may need to be compiled when you install PyAudio. If you run into compilation issues that mention PortAudio, you may have to [install some dependencies][pyaudio-install].
+The sample uses the [PyAudio][pyaudio] library to stream audio from your computer's microphone.  PyAudio depends on [PortAudio][portaudio], which may need to be compiled when you install PyAudio. 
+
+* Before proceeding [install some dependencies][pyaudio-install].
```

Comment:
Suggest:

The sample uses the [PyAudio][pyaudio] library to stream audio from your computer's microphone.  PyAudio depends on [PortAudio][portaudio], which may need to be installed separately, depending on your platform:

* Install the [pyAudio dependencies][pyaudio-install].